tests/python/gpu/test_gluon_transforms.py

```diff
@@ -145,13 +145,3 @@ def py_bilinear_resize_nhwc(x, outputHeight, outputWidth):
                             h1lambda*((1-w1lambda)*x[b][h1+h1p][w1][c] + \
                             w1lambda*x[b][h1+h1p][w1+w1p][c])
         return y
-
-    # Test with normal case 3D input int8 type
```

Comment:
Why were the two tests removed?
Others LGTM.
Thank you!

Reply:
It test ```align_core=True``` with ```align_core=False``` before, but now it's the same implementation. So it's no need.

Reply:
I see. Thank you!